website/content/en/docs/building-operators/ansible/tutorial.md

```diff
@@ -13,12 +13,13 @@ After [installing the Operator SDK CLI][install-guide] and
 new Ansible-based memcached-operator project:
 
 ```sh
-$ operator-sdk new memcached-operator --api-version=cache.example.com/v1alpha1 --kind=Memcached --type=ansible
-$ cd memcached-operator
+$ mkdir memcached-operator && cd memcached-operator
+$ operator-sdk new init --plugins=ansible --domain=example.com
+$ operator sdk create api --group=cache --version=v1alpha1 --kind=Memcached
```

Comment:
Same here. I think it will not work because will gen the empty role and the watches with a fixme. We need to use the flag.

Reply:
Updated.